<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="2" name="modelo.modelo_tests.TestStock-20160214152922" tests="3" time="0.026">
	<testcase classname="modelo.modelo_tests.TestStock" name="test100_repr" time="0.001"/>
	<testcase classname="modelo.modelo_tests.TestStock" name="test1_save" time="0.017">
		<failure message="IntegrityError('(psycopg2.IntegrityError)[164 chars]\n',) != 'Error registrando el stock.'" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/home/sebastian/proyectos/SiGeSt/src/unittest/python/modelo/modelo_tests.py", line 124, in test1_save
    self.assertEqual(result, expected)
AssertionError: IntegrityError('(psycopg2.IntegrityError)[164 chars]\n',) != 'Error registrando el stock.'

Stdout:
(psycopg2.IntegrityError) inserción o actualización en la tabla «stock» viola la llave foránea «stock_idArt_fkey»
DETAIL:  La llave (idArt)=(nArt) no está presente en la tabla «articulos».
 [SQL: 'INSERT INTO stock ("idArt", "idDep", stock) VALUES (%(idArt)s, %(idDep)s, %(stock)s)'] [parameters: {'stock': 5, 'idArt': 'nArt', 'idDep': 'nDep'}]
]]>		</failure>
	</testcase>
	<testcase classname="modelo.modelo_tests.TestStock" name="test2_save" time="0.008">
		<failure message="IntegrityError('(psycopg2.IntegrityError)[164 chars]\n',) != 'Error registrando el stock.'" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/home/sebastian/proyectos/SiGeSt/src/unittest/python/modelo/modelo_tests.py", line 129, in test2_save
    self.assertEqual(result, expected)
AssertionError: IntegrityError('(psycopg2.IntegrityError)[164 chars]\n',) != 'Error registrando el stock.'

Stdout:
(psycopg2.IntegrityError) inserción o actualización en la tabla «stock» viola la llave foránea «stock_idArt_fkey»
DETAIL:  La llave (idArt)=(nArt) no está presente en la tabla «articulos».
 [SQL: 'INSERT INTO stock ("idArt", "idDep", stock) VALUES (%(idArt)s, %(idDep)s, %(stock)s)'] [parameters: {'stock': 5, 'idArt': 'nArt', 'idDep': 'nDep'}]
]]>		</failure>
	</testcase>
	<system-out>
<![CDATA[(psycopg2.IntegrityError) inserción o actualización en la tabla «stock» viola la llave foránea «stock_idArt_fkey»
DETAIL:  La llave (idArt)=(nArt) no está presente en la tabla «articulos».
 [SQL: 'INSERT INTO stock ("idArt", "idDep", stock) VALUES (%(idArt)s, %(idDep)s, %(stock)s)'] [parameters: {'stock': 5, 'idArt': 'nArt', 'idDep': 'nDep'}]
(psycopg2.IntegrityError) inserción o actualización en la tabla «stock» viola la llave foránea «stock_idArt_fkey»
DETAIL:  La llave (idArt)=(nArt) no está presente en la tabla «articulos».
 [SQL: 'INSERT INTO stock ("idArt", "idDep", stock) VALUES (%(idArt)s, %(idDep)s, %(stock)s)'] [parameters: {'stock': 5, 'idArt': 'nArt', 'idDep': 'nDep'}]
]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
